R/baselinenowcast.R

```diff
@@ -146,7 +157,7 @@ baselinenowcast.reporting_triangle <- function(
 
   if (is.null(uncertainty_params)) {
     uncertainty_params <- estimate_uncertainty_retro(
-      reporting_triangle = data,
+      reporting_triangle = processed_data,
       n_history_delay = tv$n_history_delay,
       n_retrospective_nowcasts = tv$n_retrospective_nowcasts,
       uncertainty_model = uncertainty_model
```

Comment:
Same point below with the call to sample nowcasts, which is still using `data`. Shouldn't it be`processed_data`?

```r
nowcast_df <- sample_nowcasts(
    point_nowcast_matrix = pt_nowcast,
    reporting_triangle = processed_data,
    uncertainty_params = uncertainty_params,
    draws = draws,
    uncertainty_sampler = uncertainty_sampler,
    ...
  )
```

Reply:
fixed